docs/using_lagoon/harbor/harbor-container-settings/harbor-core.md

```diff
@@ -0,0 +1,150 @@
+# Harbor-Core Settings
```

Comment:
Are there any other setup instructions needed to get Harbor running, or will it just detect the existence of these files?

Reply:
The configuration is entirely handled by the config file and environment variables. Once those are in place, you'll have a fully functional local dev Harbor environment. In production, we have scripts which generate the ones which require a password and some logic to inject those secrets as required to make Harbor run as expected.

Reply:
I think it would be good to put this ^ in the docs!